Resources/Textures/Clothing/Mask/cluwne.rsi/meta.json

```diff
@@ -0,0 +1,18 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from tgstation at commit https://github.com/tgstation/tgstation/commit/4f6190e2895e09116663ef282d3ce1d8b35c032e, Modified by brainfood1183 (github) for ss14.",
```

Comment:
Didnt you make them? if you only took some inspiration from them you can just put "made by brainfood.. yadayada" here, if you did modify them, remember to put the exact file/folder commit link here. (same for the rest)

Reply:
I made them yea, but i obviously used inspiration from the original clown suit so i didnt think it right to just straight up say i made them. But if this is how it should be then i will change it.

Reply:
changed these now to be "Made by brainfood1183 (github) for ss14"